Update serverless pricing info (#201)
elastic/docs-content: serverless/pages/pricing.asciidoc

preview:[]

Elasticsearch is priced based on consumption of the underlying
infrastructure that supports your use case, with the performance
characteristics you need. Measurements are in Virtual Compute Units (VCUs).
Each VCU represents a fraction of RAM, CPU, and local disk for caching.

> The number of VCUs required 
> depends on the amount and rate of data sent to Elasticsearch and retained,
> as well as the number of searches and the latency you require for searches. In addition, if
> you use {ml} for inference or NLP tasks, those VCUs are also
> metered and billed.

Reviewer: ```suggestion
The number of VCUs you need is determined by:

* Volume and ingestion rate of your data
* Data retention period 
* Search query volume
* Search Power setting
* Machine learning usage (if applicable):
```

[discrete]
[[elasticsearch-billing-information-about-the-vcu-types-search-ingest-and-ml]]
== VCU types: Search, Indexing, and ML

Elasticsearch uses three VCU types:

* **Indexing:** The VCUs used to index incoming documents.
* **Search:** The VCUs used to return search results, with the latency and
queries per second (QPS) you require.
* **Machine learning:** The VCUs used to perform inference, NLP tasks, and other ML activities.

[discrete]
[[elasticsearch-billing-information-about-the-search-ai-lake-dimension-gb]]

> == Search AI Lake dimension (GB)

Reviewer: ```suggestion
== Data storage and billing
```

preview:[]

Elasticsearch is priced based on consumption of the underlying
infrastructure that supports your use case, with the performance
characteristics you need. Measurements are in Virtual Compute Units (VCUs).

> Each VCU represents a slice of RAM, CPU, and local disk for caching. 

Reviewer: ```suggestion
A VCU measures consumption of RAM, CPU, and local disk for caching.
```

I understand a lot of these edits are pre-existing text 😄  🍕

The number of VCUs required 
depends on the amount and rate of data sent to Elasticsearch and retained,
as well as the number of searches and the latency you require for searches. In addition, if
you use {ml} for inference or NLP tasks, those VCUs are also
metered and billed.

[discrete]
[[elasticsearch-billing-information-about-the-vcu-types-search-ingest-and-ml]]

> == VCU types: Search, Ingest, and ML

Reviewer 2: ```suggestion
== VCU types: Search, Indexing, and ML
```
To match with the items below?

Elasticsearch uses three VCU types:

* **Indexing:** The VCUs used to index incoming documents.
* **Search:** The VCUs used to return search results, with the latency and
queries per second (QPS) you require.
* **Machine learning:** The VCUs used to perform inference, NLP tasks, and other ML activities.

[discrete]
[[elasticsearch-billing-information-about-the-search-ai-lake-dimension-gb]]

> == Search AI Lake dimension (GB)

Reviewer: something like "How data is stored and billed"

> == Search AI Lake dimension (GB)

Reviewer 3: took your other suggestion.

> Elasticsearch stores and retains data in the <<elasticsearch-manage-project-search-ai-lake-settings,Search AI Lake>>, 
> which is charged in GB for the size of data at rest. Depending on enrichment,
> vectorization, and other activities during ingest, this size may be different
> from the original size of the source data.

Reviewer: ```suggestion
{es-serverless} projects store data in the <<elasticsearch-manage-project-search-ai-lake-settings,Search AI Lake>>. You are charged per GB of data at rest. Note that if you perform operations at ingest such as vectorization or enrichment, your storage size will differ from your original source data.
```

Suggestion to be more direct

Reviewer 2: We might want an extra explicit statement here that says which of the 2 gets billed, or reword slightly the last sentence in the suggestion

Reviewer: I just tweaked to say "stored data" in both sentences, is that clearer?

[discrete]
[[elasticsearch-billing-managing-elasticsearch-costs]]
== Managing Elasticsearch costs

You can control costs by using a lower Search Power setting or reducing the amount 
of retained data. 

* **Search Power setting:** <<elasticsearch-manage-project-search-power-settings,Search Power>> controls the speed of searches against your data. With Search Power, you can 
improve search performance by adding more resources for querying, or you can reduce provisioned 
resources to cut costs.

> * **Time series data:** By limiting the number of days of <<elasticsearch-ingest-time-series-data,time series data>> that are available for caching, 

Reviewer 2: ```suggestion
* **Time series data retention:** By limiting the number of days of <<elasticsearch-ingest-time-series-data,time series data>> that are available for caching, 
```
